hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionContext.java

```diff
@@ -170,7 +170,9 @@ public String toString() {
     Class<?> clazz = proxy.getClass();
 
     StringBuilder sb = new StringBuilder();
-    sb.append(clazz.getSimpleName())
+    sb.append("hashcode:")
+        .append(hashCode() + " ")
```

Comment:
The point of the StringBuilder is to not use +.
```
.append(hashCode())
.append(" ")
```

Reply:
fixed, thanks sir~